apps/service-portal/src/auth.ts

```diff
@@ -52,8 +51,6 @@ const SERVICE_PORTAL_SCOPES = [
   ApiScope.healthRightsStatus,
   ApiScope.healthDentists,
   ApiScope.signatureCollection,
-  NotificationsScope.read,
-  NotificationsScope.write,
```

Comment:
Doesnt service portal need something else instead or is this just being removed completely?

Reply:
yes -  these are being replaced by DocumentsScope which is already one of the imports

Reply:
Ah 😆 ofcourse !